tests/websockets/test_graphql_transport_ws.py

```diff
@@ -799,9 +800,9 @@ async def test_rejects_connection_params_not_dict(ws_raw: WebSocketClient):
     ws.assert_reason("Invalid connection init payload")
 
 
-async def test_rejects_connection_params_not_unset(ws_raw: WebSocketClient):
+async def test_rejects_connection_params_with_wrong_type(ws_raw: WebSocketClient):
```

Comment:
Maybe parametrize this with some other options? Like: a string, an integer, a list with any value in it (an empty list would work because the code is doing `payload or {}`), etc